src/Http/RequestFactory.php

```diff
@@ -76,9 +76,11 @@ public function createHttpRequest()
 			} elseif (isset($_SERVER['SERVER_PORT'])) {
 				$url->setPort((int) $_SERVER['SERVER_PORT']);
 			}
+		} else if (!empty($_SERVER['SERVER_PORT'])) {
+			$url->setPort((int) $_SERVER['SERVER_PORT']);
```

Comment:
What's the point of having port without host? Does the code solve a real world issue? If not, it should be removed.